RefactoringsExperiments/apache_repos.txt

```diff
@@ -7,4 +7,4 @@ apache/pulsar
 apache/maven-artifact-transfer
```

Comment:
let's create a folder "data" and move all files with repositories' names into it